wscript

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+import os
+
+from waflib.Configure import conf
+from waflib.Configure import ConfigurationContext
+
+APPNAME = 'waf-tools'
+VERSION = '2.34.0'
+
+@conf
+def load_external_waf_tool(ctx, name):
+    import inspect
+    this_file = inspect.getfile(inspect.currentframe())
+    path = os.path.join(os.path.dirname(this_file))
+    ctx.load([name], tooldir=[path])
+
+
+def resolve(ctx):
+
+    ctx.load_external_waf_tool('wurf_cxx_mkspec')
```

Comment:
I will try to simplify this. 
I think you can only recurse into a folder that has a wscript (or similar).
We have to `load` the tool here. We decided to have the top-level tools next to the wscript (not in subfolders).